UnitsNet/UnitDefinitions/Acceleration.json

```diff
@@ -1,20 +1,56 @@
 ﻿{
-    "Name": "Acceleration",
-    "BaseUnit": "MeterPerSecondSquared",
-    "XmlDoc": "Acceleration, in physics, is the rate at which the velocity of an object changes over time. An object's acceleration is the net result of any and all forces acting on the object, as described by Newton's Second Law. The SI unit for acceleration is the Meter per second squared (m/s²). Accelerations are vector quantities (they have magnitude and direction) and add according to the parallelogram law. As a vector, the calculated net force is equal to the product of the object's mass (a scalar quantity) and the acceleration.",
-    "Units": [
+  "Name": "Acceleration",
+  "BaseUnit": "MeterPerSecondSquared",
+  "XmlDoc": "Acceleration, in physics, is the rate at which the velocity of an object changes over time. An object's acceleration is the net result of any and all forces acting on the object, as described by Newton's Second Law. The SI unit for acceleration is the Meter per second squared (m/s²). Accelerations are vector quantities (they have magnitude and direction) and add according to the parallelogram law. As a vector, the calculated net force is equal to the product of the object's mass (a scalar quantity) and the acceleration.",
+  "Units": [
+    {
+      "SingularName": "MeterPerSecondSquared",
+      "PluralName": "MeterPerSecondSquared",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
         {
-            "SingularName": "MeterPerSecondSquared",
-            "PluralName": "MeterPerSecondSquared",
-            "FromUnitToBaseFunc": "x",
-            "FromBaseToUnitFunc": "x",
-            "Prefixes": ["Nano","Micro","Milli","Centi","Deci","Kilo"],
-            "Localization": [
-                {
-                    "Culture": "en-US",
-                    "Abbreviations": ["m/s²"]
-                }
-            ]
-        }        
-    ]
+          "Culture": "en-US",
+          "Abbreviations": [ "m/s²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InchPerSecondSquared",
+      "PluralName": "InchesPerSecondSquared",
+      "FromUnitToBaseFunc": "x/39.3700787",
+      "FromBaseToUnitFunc": "x*39.3700787",
```

Comment:
Based on http://www.kylesconverter.com/acceleration/meters-per-second-squared-to-inches-per-second-squared
FromUnitToBasefunc: `x*0.0254`
FromBaseToUnitFunc: `x/0.0254`

Reply:
Should we also include abbreviations that do not rely on the superscript?  in/s^2  Do you have a standard way of doing this?

Reply:
I'm not sure I follow. Are you saying you would like to add `in/s^2` in addition to `in/s²`, so that both can be parsed?

We recently discussed this in our work in progress contribution guideline, where we favor superscripts: https://github.com/angularsen/UnitsNet/issues/328

However, it is a point that it may be desirable to be able to parse non-superscript variants as well.
I believe you can achieve this by simply adding it: `"Abbreviations": [ "in/s²", "in/s^2" ]`
This will make the superscript one the default, used in ToString(), but also support parsing strings like `Acceleration.Parse("5.3 in/s^2")`. If that is what you meant, I think that is how you would do it in this PR.

We could consider enhancing the code generation script to automatically add `^2` and `^3` alternatives to abbreviations that use ² and ³ superscripts, so we don't have to manually duplicate this in all the .JSON files.

Reply:
Yes, I'm talking about adding additional abbreviations that might be more commonly used in coding.   Certainly having the code generation scrip automatically add the ^2 and ^3 would be ideal.  I will update the abbreviations.

Reply:
Have you made the changes that are seen above?  I'm pretty green when it comes to Git and not entirely sure how to get the most recent version of Acceleration.json locally.

Reply:
I haven't made any changes. So, since you are working on your fork's master, just continue to add more commits to that branch and push those (to your fork's master). The new commits will then show up in this PR.

Next time though, it's better to create a new branch such as `add-acceleration-units` so that you can just delete that branch when the pull request is merged. It can be confusing to figure out local vs remote master branch, as well as your fork's master vs the original repo's master. I prefer to point my local master to the original repo's master, so that I can just `git pull` to get the lastest official changes. Many ways to do this though, just ask and I'll try to help.

Reply:
I made the precision changes and did a push.